templates/app/views/checkout/registration.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= render 'spree/shared/error_messages', target: @user %>
```

Comment:
We can consider copying the `error_messages` partial from Spree. This will make it easier for a SolidusStarterFrontend user to modify it. Let me know what you think.